qiskit_nature/circuit/library/initial_states/fermionic_gaussian_state.py

```diff
@@ -0,0 +1,165 @@
+            \vdots \\
+            a_N
+        \end{pmatrix},
+
+    - :math:`a^\dagger_1, \ldots, a^\dagger_{N}` are the fermionic creation operators
+    - :math:`W` is an :math:`N \times 2N` matrix such that
+      :math:`b^\dagger_1, \ldots, b^\dagger_{N}` also satisfy the
+      fermionic anticommutation relations
+    - :math:`\lvert \overline{\text{vac}} \rangle` is the mutual 0-eigenvector of
+      the operators :math:`\{b_j^\dagger b_j\}`
+
+    The matrix :math:`W` has the block form
+
+    .. math::
+        \begin{pmatrix}
+            W_1 & W_2
+        \end{pmatrix}
+
+    where :math:`W_1` and :math:`W_2` must satisfy
+
+    .. math::
+        W_1 W_1^\dagger + W_2 W_2^\dagger = I \\
+        W_1 W_2^T + W_2 W_1^T = 0
+
+    The matrix :math:`W` is commonly obtained by calling the
+    :meth:`~.QuadraticHamiltonian.diagonalizing_bogoliubov_transform`
+    method of the :class:`~.QuadraticHamiltonian` class.
+    This matrix is used to create circuits that prepare eigenstates of the
+    quadratic Hamiltonian.
+
+    Currently, only the Jordan-Wigner Transformation is supported.
+
+    Reference: arXiv:1711.05395
+    """
+
+    def __init__(
+        self,
+        transformation_matrix: np.ndarray,
+        occupied_orbitals: Optional[Sequence[int]] = None,
+        qubit_converter: QubitConverter = None,
+        validate: bool = True,
+        rtol: float = 1e-5,
+        atol: float = 1e-8,
+        **circuit_kwargs,
+    ) -> None:
+        r"""
+        Args:
+            transformation_matrix: The matrix :math:`W` that specifies the coefficients of the
+                new creation operators in terms of the original creation and annihilation operators.
+                This matrix must satisfy special constraints; see the main body of the docstring
+                of this function.
+            occupied_orbitals: The pseudo-particle orbitals to fill. These refer to the indices
+                of the operators :math:`\{b^\dagger_j\}` from the main body of the docstring
+                of this function. The default behavior is to use the empty set of orbitals,
+                which corresponds to a state with zero pseudo-particles.
+            qubit_converter: a QubitConverter instance.
+            validate: Whether to validate the inputs.
+            rtol: Relative numerical tolerance for input validation.
+            atol: Absolute numerical tolerance for input validation.
+            circuit_kwargs: Keyword arguments to pass to the QuantumCircuit initializer.
+
+        Raises:
+            ValueError: transformation_matrix must be a 2-dimensional array.
+            ValueError: transformation_matrix must have shape (n_orbitals, 2 * n_orbitals).
+            NotImplementedError: Currently, only the Jordan-Wigner Transform is supported.
+                Please use
+                :class:`qiskit_nature.mappers.second_quantization.JordanWignerMapper`
+                to construct the qubit mapper.
```

Comment:
Maybe we can be more explicit here and refer to the actual argument `qubit_converter`.

Reply:
Done.